### PR TITLE
updated gemfile and removed test

### PR DESCRIPTION
**Pull request by askl56**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -54,19 +58,21 @@ end
...
   gem 'spring'
-
   # Coveralls, code coverage
   gem 'coveralls', require: false
+  gem 'rspec-rails'
+  gem 'spring-commands-rspec'
+  gem 'guard-rails', require: false
+  gem 'guard-rspec', require: false
+  gem 'rb-fsevent' if `uname` =~ /Darwin/
+  gem 'better_errors'
```
Comment (justanshulsharma) [inline Gemfile]: @askl56 we may also add binding_of_caller with better errors

Comment (askl56) [inline Gemfile]: Ok

**Comment by askl56** [comment]
I'm going to rebase this and merge it so the conflicts don't keep coming up. I will continue working on it.

**Comment by ur0** [comment]
Thanks @askl56. I'm not writing any new tests too.

**Comment by ur0** [comment]
This'll be the last PR which (*I think*) we'll be merging before we merge to master.

**Comment by askl56** [comment]
I'll do the rebase now.

**Comment by ur0** [comment]
:smile:

**Pull request merged**